spec/models/answer_spec.rb

```diff
@@ -0,0 +1,165 @@
+require 'rails_helper'
+
+describe Answer do
+  let(:questionnaire) {create(:questionnaire,:id => 1)}
+  let(:question1) {create(:question,:questionnaire => questionnaire,:weight=>1,:id =>1)}
+  let(:question2) {create(:question,:questionnaire => questionnaire,:weight=>2,:id =>2)}
+  let(:response_map) {create(:review_response_map,:id=>1,:reviewed_object_id => 1)}
+  let!(:response_record) {create(:response_record,:id => 1,:response_map => response_map)}
+  let!(:answer) { create(:answer,:question => question1,:response_id => 1)}
+  
+  describe "# test dependancy between question.rb and answer.rb" 
+    it { should belong_to(:question) }
+  
+  describe "#test get total score" do
+   				
+    it "returns total score when required conditions are met" do
+      # stub for ScoreView.find_by_sql to revent prevent unit testing sql db queries
+      allow(ScoreView).to receive(:find_by_sql).and_return([double("scoreview",weighted_score: 20,sum_of_weights: 5,q1_max_question_score: 4)])
```

Comment:
do I make these changes now or should I wait for the build to get stable ? It is currently failing